sympy/physics/control/lti.py

```diff
@@ -22,6 +22,7 @@
 from sympy.polys.polytools import (cancel, degree)
 from sympy.series import limit
 from sympy.utilities.misc import filldedent
+from sympy.simplify.simplify import simplify
```

Comment:
I'm not sure this is the best location for this. There may be a better place somewhere else in sympy, since it is a general tool. And this lty.py file isn't necessarily a module to hold utility functions.

Reply:
It could belong in the poylnomial package, for example.

Reply:
I think that we should move other utility functions that were added by the community like
```
 'gbt', 'bilinear', 'forward_diff', 'backward_diff',
    'phase_margin', 'gain_margin'
```
maybe utils.py could be sufficient

Reply:
If the usage of this can be more general or optimized, it could move to polytools as well later, such as `routhhurwitztools.py`

Reply:
Should I implement this function for univariate polynomials also?

Reply:
I believe that if we want to calculate the Routh array for polynomial functions, it should be added to polytools. However, this function is optimized for transfer functions. Therefore, we need to convert the transfer function to the characteristic equation first, and then apply the Routh table to it. It would be beneficial if we could pass the characteristic equation to the Routh array in polytools.

Reply:
I agree with @sylee957, it would ideally be a general tool for polynomials. I don't think that would be significant work to take that approach versus a narrow version in control.